sql/moz-fx-data-shared-prod/telemetry_derived/clients_first_seen_28_days_later_v1/query.sql

```diff
@@ -38,24 +38,20 @@ clients_first_seen_28_days_ago_with_days_seen AS (
     AND cls.submission_date = @submission_date
 )
 SELECT
-  client_id,
-  sample_id,
-  first_seen_date,
-  @submission_date AS submission_date,
-  country_code,
-  channel,
-  build_id,
-  os,
-  os_version,
-  distribution_id,
-  attribution_source,
-  attribution_ua,
-  attribution_medium,
-  attribution_campaign,
-  attribution_content,
-  attribution_experiment,
-  attribution_dltoken,
-  attribution_dlsource,
+  * REPLACE (
+    COALESCE(
+      days_seen_bits,
+      mozfun.bits28.from_string('0000000000000000000000000000')
```

Comment:
Do you do it this way to make it clear that this is a zero-activity-days bit pattern? Over e.g. `0`